File: python/bench.py
import sys
import time


def execute(target, count):
    start = time.time()
    for do_run in xrange(count):
        target.main()
    return time.time() - start


def main():
    count = int(sys.argv[1])
    use_lib = sys.argv[2]
    target = __import__(
        'script.' + use_lib,
        globals(), locals(), ['main'], -1)
    time = execute(target, count)
    print "# %s" % use_lib
    print "## %f" % time


if __name__ == "__main__":
    main()

File: java/src/Bench.java
import java.lang.reflect.Method;
import java.util.Calendar;

class Bench {
    static double execute(String target, long count) throws Exception {
        target = target.substring(0, 1).toUpperCase() + target.substring(1);

        @SuppressWarnings("unchecked")
        Class script = Class.forName(target);
        @SuppressWarnings("unchecked")
        final Method main = script.getMethod("main");
        final long start = Calendar.getInstance().getTimeInMillis();
        for (long i = 0; i < count; ++i) {
            main.invoke(null);
        }
        final long end = Calendar.getInstance().getTimeInMillis();
        return (end - start) / 1000.0;
    }

    public static void main(String[] args) throws Exception {
        final long count = Long.parseLong(args[0]);
        //final double blank = execute("blank", count);
        final double blank = 0;
        for (int i = 1; i < args.length; ++i) {
            final String target = args[i];
            final double time = execute(target, count) - blank;
            System.out.println("# " + target);
            System.out.println("## " + time);
        }
    }
}
